pyrt/light/light.py

```diff
@@ -13,11 +13,20 @@ class Light(object):
 
     """Base Light Class"""
 
-    def __init__(self, name="UNKNOWN LIGHT"):
+    def __init__(self, coef = 1.0, name="UNKNOWN LIGHT"):
         """
         Base Constructor
 
         :param name: Name of Light type, for example "PointLight".
         """
         self.name = name
         self.position = Vec3(0.,0.,0.)
+        self.coef = coef
```

Comment:
Undocumented and **coef** - is a strange name for production code(need to name according to a reference or context it is used for)